libs/s25main/world/NodeMapBase.h

```diff
@@ -61,3 +64,11 @@ void NodeMapBase<T_Node>::Resize(const MapExtent& newSize)
     nodes.clear();
     nodes.resize(prodOfComponents(newSize));
 }
+
+template<typename T_Node>
+void NodeMapBase<T_Node>::Resize(const MapExtent& newSize, const T_Node& defaultValue)
```

Comment:
Hm... This looks like it might invalidate assumptions and can only be valid if this class is final as otherwise inheritants could overwrite Resize but don't get called for this